lib/web_ui/lib/src/engine/canvaskit/font_fallbacks.dart

```diff
@@ -558,7 +583,7 @@ class NotoFont {
   NotoFont(this.name, this.approximateUnicodeRanges);
 
   String get googleFontsCssUrl =>
-      'https://fonts.googleapis.com/css2?family=${name.replaceAll(' ', '+')}';
+      '${defaultFontServerUrl}css2?family=${name.replaceAll(' ', '+')}';
```

Comment:
we should probably use `Uri.encodeQueryComponent` here rather than `replaceAll`.

cc @yjbanov

Reply:
Done. I tested it, no effect.

Uri.encodeQueryComponent('Noto Sans TC') == 'Noto Sans TC'.replaceAll(' ', '+') // true

Reply:
one example where it would have an effect is if the font name contained a slash, for example. Or anything non-ASCII.